src/base/static/map/index.js

```diff
@@ -0,0 +1,520 @@
+/* global $ Shareabouts */
```

Comment:
I don't think it should be a `lib` or a `util` really. It seems too important for that. Could it be an `organism` as is?

Reply:
organisms are components. Right now, it's not a component, but eventually it will be. Until it is, we should put this with our other `lib`'s or `util`'s.

Reply:
Would it make sense to put the map module in `map-providers`, along with the factories? Otherwise I think it would make sense to put it with `libs` rather than `utils`.

Reply:
There are probably multiple ways this can be organized. I would do this:

 * move `libs` to `legacy-libs`
 * make a new `libs` dir
 * make a new `libs/maps` dir
 * have the file `libs/maps/index.js`, and something like `libs/maps/mapbox-provider.js`, and `libs/maps/abstract-provider.js` (notice how everything under `libs/maps` is flattened)

Reply:
That sounds good to me.